pkg/process/config/data_scrubber.go

```diff
@@ -13,10 +13,10 @@ import (
 
 var (
 	defaultSensitiveWords = []string{
-		"password", "passwd", "mysql_pwd",
-		"access_token", "auth_token",
-		"api_key", "apikey",
-		"secret", "credentials", "stripetoken"}
+		"*password*", "*passwd*", "*mysql_pwd*",
+		"*access_token*", "*auth_token*",
+		"*api_key*", "*apikey*",
+		"*secret*", "*credentials*", "stripetoken"}
```

Comment:
is there a reason stripetoken isn't wrapped as well ?

Reply:
Yeah - I think it's a very specific case that likely doesn't need wildcards by default